frontend/src/views/CanteenEditor.vue

```diff
@@ -202,6 +202,39 @@
         </v-col>
       </v-row>
     </div>
+    <v-divider class="my-10" v-if="!isNewCanteen"></v-divider>
+
+    <v-dialog v-model="deletionDialog" width="500" v-if="!isNewCanteen">
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn color="red" outlined v-if="!isNewCanteen" v-bind="attrs" v-on="on">
+          <v-icon small class="mr-2">mdi-alert</v-icon>
+          Supprimer cette cantine
+        </v-btn>
+      </template>
+
+      <v-card class="text-left">
+        <v-card-title class="font-weight-bold">
+          Êtes-vous sur de vouloir supprimer cette cantine ?
+        </v-card-title>
+
+        <v-card-text>
+          Cette action est irréversible. Vous perdrez les données de cette cantine ainsi que celles des diagnostics lui
```

Comment:
```suggestion
          Vous perdrez les données de cette cantine ainsi que celles des diagnostics lui
```
Pour l'instant c'est pas le cas que cette action est irréversible